.github/workflows/build.yaml

```diff
@@ -26,6 +26,7 @@ jobs:
       LATEST_TAG: latest
       # this needs to be updated on every release
       SOLR_CURR_VERSION: 6.6.11
+      DOCKERFILE: "Dockerfile.amd"
```

Comment:
action is not running because it is waiting for ubuntu-18
should be ubuntu latest, check any other repo

Reply:
okay updated it to latest